Update from v4.3.x to v4.4 (#1870)
ezsystems/developer-documentation: docs/update_and_migration/from_4.3/update_from_4.3.md

---
description: Update your installation to the latest v4.4 version from v4.3.x.
---

# Update from v4.3.x to v4.4

This update procedure applies if you are using the newest v4.3 installation.

There are two routs you can take.
The first one is for you, if you do not use old commerce packages, then you can proceed with removing them.

> The second one is for you if you use commerce packages and would like to continue to use them, even though they have deprecated since v4.4.

Reviewer: Can we add explanation why there are two routes the begging ? Something like "ibexa/commerce-* packages are deprecated since 4.4.0 ..."

Author: We can add some additional explanation, the landing page with cards currently looks like that: 
![image](https://user-images.githubusercontent.com/35031941/215766397-69dd22c3-0ba0-47bc-ab83-b624a62cbdfa.png)

---
description: Update your installation to the latest v4.4 version from v4.3.x.
---

# Update from v4.3.x to v4.4

This update procedure applies if you are using the newest v4.3 installation.

> In this release, we deprecated all Commerce packages in Ibexa DXP and plan to remove them in v5.

Reviewer 2: ```suggestion
This release deprecates all Commerce packages in Ibexa DXP. They will be removed in v5.
```

Until that time, they will be maintained by Ibexa with fixes, including security fixes, but they won't be further developed.
Old packages are replaced by [the all-new Ibexa Commerce packages](ibexa_dxp_v4.4.md#all-new-ibexa-commerce-packages).

> For that reason, there are two update routs you can take.

Reviewer 2: ```suggestion
For that reason, there are two update routes you can take.
```

> The first one is for you, if you do not use Commerce packages, then you can proceed with removing them.
> The second one is for you if you do use deprecated Commerce packages and would like to continue to use them for the time being.

Reviewer 2: How about:
```suggestion
A. If you do not use Commerce functionalities, you can proceed with removing them.
B. If you use Commerce functionalities based on the deprecated packages, you can continue to use them for the time being.
```
I'd focus on "functionalities" first, not packages
